stacker/plan.py

```diff
@@ -191,7 +196,8 @@ def _single_run(self):
                 )
                 continue
 
-            if not step.done and self._watch_func and step_name not in self._watchers:
+            if not step.done and self._watch_func and \
```

Comment:
i hate you and your weird style.

i would do this as:

```py
if (
    not step.done and
    self._watch_func and
    step_name not in self.watchers
):
```

i abhor `\` usage